4주차/[Lv.2]_[3차]_방금그곡/안혜수.js

```diff
@@ -0,0 +1,18 @@
+function solution(m, musicinfos) {
+  var answer = [0, '(None)'];
```

Comment:
```suggestion
  let answer = [0, '(None)'];
```
`var`보다는 `let` 변수로 쓰시는 것이 더 좋을 것 같습니다!! 🙋🏻‍♂️